guides/doc-Administering_Red_Hat_Satellite/topics/configuring-idm-authentication-on-satellite-server.adoc

```diff
@@ -77,6 +77,18 @@ Replace _OTP_ with the one-time password provided by the {FreeIPA} administrator
 +
 The installer is dependent on packages which, on Red{nbsp}Hat Enterprise{nbsp}Linux{nbsp}7, are in the optional repository `rhel-7-server-optional-rpms`.
 
+ifndef::satellite[]
+. If {ProjectServer} is running on Debian, ensure that the hostname is set to the FQDN:
```

Comment:
technically, this is not *specific* to Debian, but EL by default does have the hostname as FQDN, while Debian does not.

Reply:
Perhaps not everyone will understand this?
```suggestion
ifndef::satellite[]
. If {ProjectServer} is running on Debian, ensure that the hostname is set to the FQDN (including the domain):
```

Reply:
Or "… FQDN (not only the short name)"? Feels weird to "duplicate" "domain".

Reply:
Should we spell out FQDN perhaps?
```suggestion
ifndef::satellite[]
. If {ProjectServer} is running on Debian, ensure that the hostname is set to a Fully Qualified Domain Name. This means the short name is insufficient:
```

Reply:
Looking at other snippets in the docs, I think I'd go with this:

```suggestion
ifndef::satellite[]
. If {ProjectServer} is running on Debian, ensure that the hostname is set to the fully qualified domain name (FQDN), the short name is not sufficient:
```

Reply:
This is good, if you change the comma after FQDN to a semicolon, it is perfect.